modules/lang-painless/src/yamlRestTest/resources/rest-api-spec/test/painless/70_execute_painless_scripts.yml

```diff
@@ -189,7 +189,6 @@ setup:
             document:
               point: "30.0,40.0"
             index: "my-index"
-  - match: { result: [ 3176939252927413179 ] }
```

Comment:
Shod this have a new version?

Reply:
Good question! I wasn't sure what to do here. I mimicked what happens in for runtime fields which gives the format `30.0, 40.0` as a String. But because they go through double transformations it ends up with `29.999997, 39.9999987` kind of values. Given that this is a String, but with double precision, do you have any suggestions for how to test this appropriately? I also test this in the single node tests by splitting the String and doing close to kind of double checks.

Reply:
Added close_to here for all these tests.